docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Drawer.htm

```diff
@@ -221,11 +221,13 @@ <h3>Summary</h3>
       &lt;li class=&quot;ui-drawer-sub-list&quot;&gt;&lt;a href=&quot;#&quot;&gt;Sub item 1&lt;/a&gt;&lt;/li&gt;
    &lt;/ul&gt;
 &lt;/div&gt;
-&lt;script&gt;
-   var drawerElement = document.getElementById(&quot;leftdrawer&quot;),
-       drawer = tau.widget.Drawer(drawerElement);
-   drawer.close();
-&lt;/script&gt;
+</pre>
+<p>JS code:</p>
```

Comment:
Global Suggestion : How about adding "\<p>HTML code:\</p>" as "\<p>JS code:\</p>" added?

If you want to keep current state, I'll merge this PR.